### DISCUSSION TITLE
Please provide syntax for credentials.json

### DISCUSSION THREAD

**Discussion by dvandemaele** [ether/etherpad-lite#5344] [Q&A] [answered]
Hello,

Several places suggest creating and using a credentials.json file for 'credentials'.

Are these for database or user credentials?

Please provide syntax examples for each.

**Comment by gwire** [comment, marked as answer]
My understanding is that `credentials.json` has the same format as `settings.json` - ie `credentials.json` will be loaded, and then `settings.json` will override it. So if you specify database settings in credentials, but not in settings, then the values in credentials will be used.

**Reply by SamTV12345** [reply]
Correct. You can put it exactly where settings.json is. It is just a measure to separate concerns. Settings should not be mixed with credentials.
